library/cohesity_job.py

```diff
@@ -532,7 +533,7 @@ def start_job(module, self):
         payload['runType'] = "k" + self['runType']
 
         data = json.dumps(payload)
-        # module.exit_json(output=data)
+        # module.exit_json(output=payload)
```

Comment:
why are we changing the output type , can you give an example of how it was previously vs now ?